mmv1/third_party/terraform/services/firebaseappcheck/test-fixtures/private-key.p8

```diff
@@ -0,0 +1,3 @@
+-----BEGIN PRIVATE KEY-----
+NotAValidKey
```

Comment:
Just want to flag that we've run into problems in the past with APIs adding validation for things like private keys, which then causes test failures that have to be addressed - if that's something that might happen here it could make sense to provide a valid private key that is generated specifically for these tests and not used elsewhere.

Reply:
Requesting for a valid key at the moment. I'll update the thread when ready.